steward/src/cellars/aave_v2_stablecoin.rs

```diff
@@ -36,14 +36,26 @@ pub fn get_encoded_call(function: Function, cellar_id: String) -> Result<Vec<u8>
             let call = ClaimAndUnstakeCall {};
             Ok(AaveV2StablecoinCellarCalls::ClaimAndUnstake(call).encode())
         }
-        EnterPosition(_) => {
+        EnterPosition(params) => {
+            let assets = string_to_u256(params.assets)?;
             log_cellar_call(
                 CELLAR_NAME,
-                &EnterPositionCall::function_name(),
+                &EnterPositionWithAssetsCall::function_name(),
```

Comment:
Looking at the fact that both types of enter/exit functions, with or without assets specified, are external and available, why are we only exposing one of the two?